### PR TITLE
Added '-jsx' to docs

### PR DESCRIPTION
**Pull request by lynnaloo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -169,12 +169,12 @@ grunt.initConfig({
...
 #### Specifying JSHint options and globals
 
-In this example, custom JSHint options are specified. Note that when `grunt jshint:uses_defaults` is run, those files are linted using the default options, but when `grunt jshint:with_overrides` is run, those files are linted using _merged_ task/target options.
+In this example, custom JSHint options are specified. Note that when `grunt jshint-jsx:uses_defaults` is run, those files are linted using the default options, but when `grunt jshint-jsx:with_overrides` is run, those files are linted using _merged_ task/target options.
 
 ```js
 // Project configuration.
 grunt.initConfig({
-  jshint: {
+  jshint-jsx: {
```
Comment (akbiggs) [inline README.md]: Missing the quotes here.

File: README.md [hunk 2]
```diff
@@ -210,7 +210,7 @@ You can toggle it by prepending `-` to the warning id as an option:
 
 ```js
 grunt.initConfig({
-  jshint: {
+  jshint-jsx: {
```
Comment (akbiggs) [inline README.md]: see above

**Comment by demmer** [comment]
@lynnaloo thanks for the PR. Happy to merge when you fix the problems identified by @akbiggs.

File: README.md (hunk 1 above)
Comment (lynnaloo) [inline README.md]: Good catch! Let me fix that really quick :)

**Comment by Enet4** [comment]
I'm glad I thought of checking the repository. I was having some trouble integrating the task, simply due to a misleading documentation. Hope to see this merged soon, or more people might stumble upon the same issue. ;)

**Comment by demmer** [comment]
I was waiting on @lynnaloo to fix the quoting problems -- but those are trivial to adjust so I'll just merge as-is and then fix it up myself.

**Pull request merged**